IOS/SwiftSeedProject/SwiftSeedProject/Infrastructure/Network/Protocols/RestClientManagerProtocol.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ApiManagerProtocol.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 6/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+protocol RestClientManagerProtocol {
+    func sendRequest(target: Target, parameters: [String : AnyObject]?, headers: [String : String]?) -> RestClientRequestProtocol
+}
+
+extension RestClientManagerProtocol {
+    func sendRequest(target: Target) -> RestClientRequestProtocol {
+        return sendRequest(target: target, parameters: nil, headers: nil)
+    }
+    
+    func sendRequest(target: Target, parameters: [String : AnyObject]?) -> RestClientRequestProtocol {
```

Comment:
maybe we can use this function always. Also, we can change this: 
func sendRequest(target: Target, parameters: [String : AnyObject]?) -> RestClientRequestProtocol
to this:
func sendRequest(target: Target, parameters: [String : AnyObject]? = nil ) -> RestClientRequestProtocol